homeassistant/components/avocent_dpdu/coordinator.py

```diff
@@ -0,0 +1,56 @@
+"""DataUpdateCoordinator for Avocent DPDU."""
+
+from datetime import timedelta
+from typing import Any
+
+from avocentdpdu.avocentdpdu import AvocentDPDU
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.debounce import Debouncer
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN, LOGGER
+
+
+class AvocentDpduDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching Avocent DPDU data from single endpoint."""
+
+    _connected: bool = False
+
+    def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
```

Comment:
```suggestion
    config_entry: ConfigEntry

    def __init__(self, hass: HomeAssistant) -> None:
```
Now you can use `self.config_entry` after calling the super constructor